docs/overview/migrating-react-native.md

```diff
@@ -5,35 +5,103 @@ sidebar_label: Migrating react-native
 original_id: migrating_react_native
 ---
 
-
-
 ReNative supports react-native projects pretty much out of the box in terms of your application code
 
 Due to variety of possibilities / complexity fo react-native projects (ie plugins, custom build scripts and so on) there might be some extra work required to reconnect existing plugins
 
 ---
+
 ## Migration Guide
 
 1. Install rnv
 2. Create new project `rnv new` (pick @rnv/template-starter)
-3. follow rest of the steps
-4. After first `rnv run ...` check out the codebase and pay attention to:
+3. Follow rest of the steps of new project installation.
+4. After first `rnv run ...` check out the codebase.
+
+### ReNative Project Folder Structure
+For more information about the project structure, please visit this <a href="/docs/overview/architecture#build-process">link</a>.
+
+The `src/` folder contains the source content for your application. The initial structure of the `src/` folder includes the following subfolders:
+
+-   `app`: This folder contains the main files for your application.
+-   `entry`: This folder allows you to configure the launch settings of your app. It is intended for advanced configuration and customization.
+-   `pages`: This folder enables you to create individual pages. This is only used for web development.
+
+:::info
+You are not restricted to the three specified folders. Within the `src/` directory, you have the flexibility to create your own folders to organize styles, components, and any other necessary resources.
+:::
+
+If you also installed some react native plugins in your project make sure to move them to `./renative.json` config file under `plugins` object. 
+
+Renative uses standard react native plugins so plugin names will stay same!
+
+ReNative does not utilize autolinking as React Native does. 
+Therefore, it's important to manually move and configure packages in your renative.json file. 
+Proper handle of dependencies:
 
--   `./index.ios.js` file
--   `./index.android.js` file
--   `src/` folder
+- Transfer the necessary packages to the renative.json file.
+    For example:
+    ```
+    "plugins": {
+        "react-native": {
+            "version": "0.73.4",
```

Comment:
```suggestion
```
Maybe better to exclude this as it would probably not work with the latest react native version if there are changes.